scenes/IngameScores.gd

```diff
@@ -0,0 +1,31 @@
+extends VBoxContainer
+
+const score_viewer := preload("res://scenes/performance/ingame_user_score_viewer.tscn")
+
+func reload_users(ingame_users : Dictionary) -> void:
+	for c in get_children():
+		c.name = "__tmp"
+		c.queue_free()
+	load_users(ingame_users)
+
+func load_users(ingame_users : Dictionary) -> void:
+	for k in ingame_users:
+		var n = get_node_or_null(k)
+		if !n:
```

Comment:
Please use descriptive names as much as possible.

for child in get_children()

for ingame_user in ingame_users

var node = get_node_or_null(ingame_user)